src/ServerTests/install-nginx.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-curl -sSL http://nginx.org/download/nginx-1.8.0.tar.gz | tar zxfv - -C /tmp && cd /tmp/nginx-1.8.0/
```

Comment:
Do we need this script for anything?

Reply:
I didn't see this script being used anywhere. Also, this file is removed in 2.2. and master. I think it's safe to remove.